src/sumOfSquares.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const { square } = require('./square');
+const { sum } = require('./sum');
+
+function sumOfSquares(x, y) {
+  const a = square(x);
+  const b = square(y);
+
+  const c = sum(a, b);
+
+  return c;
```

Comment:
Too many unnecessary variables.
Calculations are simple enough to return value right away.
```suggestion
  return sum(square(x), square(y));
```